app/src/main/res/xml/fragment_navigation_view_preferences.xml

```diff
@@ -19,4 +19,10 @@
         android:entries="@array/unit_type_array"
         android:entryValues="@array/unit_type_values_array"
         android:defaultValue="-1"/>
+    <ListPreference
+        android:defaultValue="1"
+        android:entries="@array/route_profile_array"
+        android:entryValues="@array/route_profile_values_array"
+        android:key="route_profile"
```

Comment:
you could add this to strings.xml as `route_profile_key` like the other list preferences do